codewars/converter.rb

```diff
@@ -0,0 +1,9 @@
+MILE_TO_KM = 1.609344
```

Comment:
`1.609344` => `1.609_344`

Reply:
use underscore to separate 3 digits it's more readable

Reply:
I think naming is not correct. For me it's better `KM_IN_MILE` and `LITERS_IN_GALLON`